Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/Schedule.kt

```diff
@@ -8,6 +10,7 @@ import java.util.*
  */
 class Schedule {
 
+    @Parcelize
```

Comment:
This is so interesting! I remember using a parcelable library back in 2016 didn't realize kotlinx got it

Reply:
Yeah it's super helpful. :)